### ISSUE TITLE
rffit creation and response y used

### ISSUE THREAD

**Issue by andrejadd** [chengwang88/deltavina#1]
Dear chengwang88, 

thanks for the tool, I was going to test the feature set and response with another method. 

I noticed that the *rffit* model of `randomForest` is provided and can be used to make new predictions. However, I would be also interested in the part that creates *rffit* in particular the y response variable (output value). In the associated paper from 2017 [1], it is stated that this is `delta pKd(train)`, which is equal to `pKd(train) - pKd(VINA)`. If `pKd(VINA)` is the score from Vina, could you please let me know what `pKd(train)` corresponds to? I have trouble tracking down this variable. I'd like to make sure to use the same response as yours such that I can make a fair comparison. Many thanks in advance.

[1] Cheng Wang 1 and Yingkai Zhang (2017). Improving Scoring-Docking-Screening Powers of Protein-Ligand Scoring Functions using Random Forest.

**Comment by andrejadd** [comment]
I just found out from the paper that `pKd(train)` is the experimental binding affinity. Would it be anyway possible to provide the full training data set with the response variables and labels. Thanks.
